src/headless/tasks/inasafe_wrapper.py

```diff
@@ -281,3 +286,46 @@ def check_broker_connection():
     :return: True
     """
     return True
+
+
+@app.task(
+    name='inasafe.headless.tasks.push_to_geonode',
+    queue='inasafe-headless-geonode')
+def push_to_geonode(
+        layer_uri,
+        geonode_url=REALTIME_GEONODE_URL,
```

Comment:
Wait a minute. I thought you are taking this from settings, then why you also accepts url?
This would mean you are passing password in between unsecure connections.

Reply:
Ok, move back to the setting.